### PR TITLE
#152 fixing CSS Layout cumulative shift for search results

### LINKED ISSUE #152: Improve CSS for the search functionality & LHS: reduce layout shift as per issue 149
image

Please check the link below - 

https://main--24life--hlxsites.hlx.page/search?q=san+francisco

### PR DESCRIPTION
**Pull request by meejain**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #152 #149 

Test URLs:
- Before: https://main--24life--hlxsites.hlx.live/search?q=francisco
- After: https://issue152-css-layout--24life--hlxsites.hlx.live/search?q=francisco

### CODE REVIEW

File: scripts/lib-franklin.js [hunk 1]
```diff
@@ -264,6 +264,7 @@ export async function fetchPlaceholders(prefix = 'default') {
  */
 export function decorateBlock(block) {
   const shortBlockName = block.classList[0];
+  console.log(shortBlockName);
```
Comment (synox) [inline scripts/lib-franklin.js:267]: Unexpected console statement

File: blocks/search-results/search-results.js [hunk 2]
```diff
@@ -92,8 +90,9 @@ function noResults(resultsDiv) {
  `;
   resultsDiv.parentNode.querySelector('.block.search-results.card-container.three-columns .results-div').classList.add('no-results-div');
 }
-
-function displayNextEntries(iterator, numInitialLoadedArticles, loadMoreContainer, resultsDiv) {
+// eslint-disable-next-line
```
Comment (synox) [inline blocks/search-results/search-results.js:93]: only disable specifically

**Review by synox — CHANGES_REQUESTED** [review]
This does not yet address the layout shift of #149

File: blocks/search-results/search-results.css
Comment (synox) [inline blocks/search-results/search-results.css:1]: Please also set a clearer title for the PR, you can usually copy from the issue

File: scripts/lib-franklin.js (hunk 1 above)
Comment (meejain) [reply scripts/lib-franklin.js:267]: Fixed

File: blocks/search-results/search-results.js (hunk 2 above)
Comment (meejain) [reply blocks/search-results/search-results.js:84]: fixed

**Comment by meejain** [comment]
After making the changes in the nav flex - we could see the change in the cumulative lay out shift

image

**Comment by meejain** [comment]
Further search queries - https://issue152-css-layout--24life--hlxsites.hlx.live/search?q=game+francisco
image

**Review by synox — APPROVED** [review]

**Pull request merged**